Sources/protoc-gen-swift/FileGenerator.swift

```diff
@@ -370,5 +370,21 @@ class FileGenerator {
             p.outdent()
             p.print("]\n")
         }
+
+        generateVersionCheck(printer: &p)
```

Comment:
That's a good reason for doing it, but it's also counter to #35—putting stuff the user is most interested in (in the typical case) up top.

The protocol could be collapsed into a one-liner, but I wouldn't want to lose the block of comments that actually explains what it's doing.